bionic/flow.py

```diff
@@ -230,7 +233,7 @@ def build(self):
         self._state = state.touch()
         return flow
 
-    def declare(self, name, protocol=None):
+    def declare(self, name, protocol=None, docstring=None):
```

Comment:
Do you think it's better to say `docstring` or `doc`?  The former is clearer but the latter is shorter and still pretty clear.  Genuine question, I could go either way.  WDYT?

Reply:
I would vote `docstring` here :) but it's an easy change either way